styles.css

```diff
@@ -433,3 +435,161 @@ li {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 600px) {
+  .menu {
+    display: none;
+  }
+
+  .navigation {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 32px;
+    padding: 0;
+    height: 916px;
+    left: 1139px;
+    top: 16px;
+  }
+
+  .item-navigation {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    padding-right: 90px;
+    text-align: center;
+    color: #344563;
+    gap: 32px;
+  }
+
+  .mobile-portfolio {
+    position: absolute;
+    left: 0;
+    right: 0;
+    display: flex;
+    width: 100%;
+    height: 50px;
+    justify-content: space-between;
+    align-items: center;
+    padding: 14px 24px;
+  }
+
+  .first-page {
+    min-width: 375px;
+    padding-top: 90px;
+    padding-bottom: 80px;
+    padding-left: 260px;
+    padding-right: 260px;
+    height: 846px;
+    background: #fff;
+    background-image: url("image/Header bg.svg");
+    background-repeat: no-repeat;
+    background-position-y: 70px;
+    background-size: 100%;
+    border-bottom-left-radius: 6rem;
+  }
+
+  .work {
+    padding-top: 10%;
+  }
+
+  .project {
+    background: #fff;
+    border: 0.2rem solid #e5e5e5;
+    border-radius: 20px;
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10%;
+    left: 0;
+    right: 0;
+    width: 1056px;
+    height: 456px;
+    margin-right: 142px;
+    margin-left: 142px;
+  }
```

Comment:
- [ ] In the mobile version the image should be shown first then there should be cards also fix this issue. Also fix the color of the first list in all the cards
### Your design
![your](https://user-images.githubusercontent.com/81727988/213684126-8e923242-dad6-4afe-8c6c-9aab6890bab6.jpg)